app/services/inline_uploads.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_dependency "pretty_text"
+
+class InlineUploads
+  def self.process(markdown)
```

Comment:
maybe support: `InlineUploads(markdown, on_missing: lambda{ |url| 77  }`

every missing upload gets upload_id 77, this is super flexible cause we can inject how to behave with unknown url from the outside

Reply:
so we can use a google search to try and find the image and then create upload, finally giving it an id. or search on disk ... the or in the case of broken image inline a broken image